app/views/users/_edit.haml

```diff
@@ -5,242 +5,241 @@
+
+    %hr
+    .row
+      #account_data.col-lg-7
+        .row
+          %h3.col-md-12= t('.export_data')
+          .col-md-6.form-group
+            - if current_user.exporting
+              .export-in-progress= t('.export_in_progress')
+            - elsif current_user.export.present?
+              = link_to t('.request_export_update'), export_profile_user_path, method: :post, class: "btn btn-default btn-block"
+              .small-horizontal-spacer
+              = link_to t('.download_export'), download_profile_user_path, class: "btn btn-success btn-block"
+              .small-horizontal-spacer
+              %h6
+                = t('.last_exported_at', timestamp: current_user.exported_at)
+            - else
+              = link_to t('.request_export'), export_profile_user_path, method: :post, class: "btn btn-default btn-block"
+
+          .col-md-6.form-group
+            - if current_user.exporting_photos
+              .export-in-progress= t('.export_photos_in_progress')
+            - elsif current_user.exported_photos_file.present?
+              = link_to t('.request_export_photos_update'), export_photos_user_path, class: "btn btn-default btn-block"
+              .small-horizontal-spacer
+              = link_to t('.download_export_photos'), download_photos_user_path, method: :post, class: "btn  btn-success btn-block"
+              .small-horizontal-spacer
+              %h6
+                = t('.last_exported_at', timestamp: current_user.exported_photos_at)
+            - else
+              = link_to t('.request_export_photos'), export_photos_user_path, method: :post, class: "btn btn-default btn-block"
+
+      .col-lg-5
+        %h3.text-right
+          = t('.close_account_text')
+
+        - if mobile
+          = form_for "user", url: user_path, html: { method: :delete } do |f|
             = f.error_messages
             .form-group
-              = f.text_field :email, value: @user.unconfirmed_email || @user.email, class: "col-md-7 form-control"
-            .form-group
-              .clearfix= f.submit t('.change_email'), class: "btn btn-primary pull-right"
-          - if @user.unconfirmed_email.present?
-            %div= t('.email_awaiting_confirmation', email: @user.email, :unconfirmed_email => @user.unconfirmed_email)
+              .clearfix= f.label :close_account_password, t("users.edit.current_password"), for: :close_account_password,
+                      class: "pull-right", style: "padding-top: 9px;"
+              .small-horizontal-spacer
+              .clearfix= f.password_field :current_password, id: :close_account_password, class: "form-control"
+              %p
+              .clearfix= f.submit t('users.edit.close_account_text'), class: "btn btn-danger pull-right",
+                      id: "close_account_confirm", data: { confirm: t("are_you_sure_delete_account") }
+
+        - if not mobile
```

Comment:
I didn't touch this code at all, I only reindented it that's why it is shown as modified.